packages/api/README.md

```diff
@@ -31,18 +32,19 @@ Due to this dynamic nature, this API departs from traditional APIs which only ha
 Installation -
 
 ```
-npm install --save @polkadot/api
+yarn add @polkadot/api
 ```
 
 Subscribing to blocks via Promise-based API -
 
 ```javascript
-import { ApiPromise } from '@polkadot/api';
+import Api from '@polkadot/api/promise';
```

Comment:
Any reason this is not the original import? (This has been done in a couple of places now). This example use is actually great since it shows the 2 side-by-side, so being a bit more explicit as it was is a bit cleaner and clearer.

Reply:
I just changed the naming for consistency based on this review comment https://github.com/polkadot-js/api/pull/292#discussion_r238654837.

I agree that it would be better to show more explicitly here though since users can compare them side-by side.

I've pushed a commit reverting back